projects/packages/identity-crisis/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "jetpack-identity-crisis",
-	"version": "0.2.5",
+	"version": "0.2.6-alpha",
```

Comment:
What is the rationale for this one package being bumped where the others have not?

Reply:
I'm also wondering this. I went back to the PR that last updated the node version, https://github.com/Automattic/jetpack/pull/20827, and see some packages bumped their patch version while others appear to have not. I'm not sure what's the reason for this yet.

Reply:
Other packages are already on the appropriate -alpha versions and so do not need a bump. This one happens to not have been touched by any other PR since the last time it was released, so it hadn't been bumped yet.

Reply:
Thanks! Makes sense 👍

Reply:
FYI, it turns out that one other package does need a bump: packages/jitm doesn't have a package.json, but it does have a version number in `projects/packages/jitm/src/class-jitm.php`.

Maybe we output too much debugging information so [the error in the console](https://github.com/Automattic/jetpack/runs/3545087975?check_suite_focus=true#step:4:431) is hard to find. It's a little easier if you look at the bottom of [the workflow summary page](https://github.com/Automattic/jetpack/actions/runs/1213484939) or in [the file list](https://github.com/Automattic/jetpack/pull/20990/files#file-projects-packages-jitm-src-class-jitm-php-L23).